service-graph/design.md

```diff
@@ -1,23 +1,35 @@
 # Service Graph
 
 ## Service Representation Colors
-Nodes in the service graph are colored according to their health. Borders are 1px wide for unselected nodes and 3px for selected nodes.
+Nodes in the service graph are colored according to their health. Borders are 1px wide for unselected nodes and 3px for selected nodes. Graph edges are colored similarly, with 1px lines when unselected, and 3px lines when selected. There is no separate hover state for graph edges, but items not directly connected to the hovered edge should be blurred into the background.
 
-### Healthy Nodes
+### Healthy
 ![healthy nodes](img/node-status-default.png)
-- Grey Stroke: #bbbbbb (pf-black-400)
+![healthy edges](img/edge-status-healthy.png)
+- Grey Stroke: #8b8d8f (pf-black-500)
 - White Fill: #ffffff (pf-white)
 - Blue Stroke: #39a5dc (pf-blue-300)
 - Blue Fill: #def3ff (pf-blue-50)
+- Green Stroke: #3f9c35 (pf-green-400)
 
-### Warning Nodes
+### Warning
 ![warning nodes](img/node-status-warning.png)
+![warning edges](img/edge-status-warning.png)
 - White Fill: #ffffff (pf-white)
 - Orange Stroke: #ec7a08 (pf-orange)
 - Orange Fill: #fdf2e5
 
-### Error Nodes
+### Error
 ![error nodes](img/node-status-error.png)
+![error edges](img/edge-status-error.png)
 - White Fill: #ffffff (pf-white)
 - Red Stroke: #cc0000 (pf-red)
 - Red Fill: #ffe6e6
+
+### Idle
+![idle edges](img/edge-status-idle.png)
+- Grey Stroke: #8b8d8f (pf-black-500)
+
+### Unused
+![unused edges](img/edge-status-unused.png)
+- Grey Stroke: #8b8d8f (pf-black-500)
```

Comment:
What is the use case for "unused"?

Reply:
In the current legend, that's labeled with "no traffic yet"

Reply:
Isn't Idle == no traffic?
If unused means no traffic, what idle means then?

Reply:
Edges to an unused node have no traffic (i.e. the dotted lines).  They exist only when there are multiple versions and we group it with the other version nodes.   Having the same color for 
"idle"  edges (i.e. an edge that has shown traffic for other time ranges, but not for the current one) and unused edges (has never seen traffic), makes sense.

As for the legend, I think this will be a high priority for Sprint 10, and it should be replaced with a  dialog with much more detailed descriptions.